mod/consensus-types/pkg/types/block.go

```diff
@@ -87,7 +87,7 @@ func (b *BeaconBlock) NewFromSSZ(
 		block = &BeaconBlock{}
 		return block, block.UnmarshalSSZ(bz)
 	case version.DenebPlus:
-		panic("unsupported fork version")
+		panic(ErrForkVersionNotSupported)
```

Comment:
just found out this error was not consolidated and fixed it. It was used in one of the relevant callers